EQL-Client-Project/src/components/header/dropdown/DropDown.style.tsx

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+export const DropDownContainer = styled.section`
+  position: absolute;
+
+  display: flex;
+  justify-content: flex-start;
+
+  width: 100%;
+  height: 43.9rem;
+  padding-top: 2.4rem;
+  padding-left: 3.1rem;
```

Comment:
요거는 이렇게 합쳐줄 수도 있겠네요~
```suggestion
  padding: 2.4rem 0 0 3.1rem;
```

Reply:
적극 활요하겠습니당